spec/mongoid/query_cache_spec.rb

```diff
@@ -123,6 +123,27 @@
         end
       end
     end
+
+    context "when cache is enabled and second batch of items is requested" do
+
+      before do
+        Mongoid::QueryCache.enabled = true
+
+        5.times do |i|
+          Band.create(views: i % 10)
+        end
+      end
+
+      it "no error raised getting next batch from cache" do
```

Comment:
@DiannaHohensee You can change this string to 

'does not raise an error when requesting the second batch'